prow/pubsub/subscriber/subscriber.go

```diff
@@ -478,6 +478,12 @@ func tryGetCloneURIAndHost(pe ProwJobEvent) (cloneURI, host string) {
 		return "", ""
 	}
 
+	// If the Refs struct already has a populated CloneURI field, use that
+	// instead.
+	if refs.CloneURI != "" {
+		return refs.CloneURI, strings.TrimSuffix(refs.Repo, refs.CloneURI)
```

Comment:
also I would suggest return refs.org for host if it's present, otherwise empty string might be more preferable than this, for example this wouldn't work with `git@github.com:kubernetes/test-infra.git` as cloneURI, and might not work for a different source control system